cloudsync/sync/manager.py

```diff
@@ -999,6 +1007,9 @@ def _handle_dir_delete_not_empty(self, sync, changed, synced):
                     all_synced = False
                     break
             if all_synced:
+                # from https://github.com/AtakamaLLC/cloudsync/pull/256/files
+                # This is bad, because we don't even check the file system to ensure that we are actually synced,
```

Comment:
this issue was fixed in https://github.com/AtakamaLLC/cloudsync/commit/03a617d9f27a10fe2296e6fa80f4c0fb67b4d637